src/aap_eda/api/serializers/activation.py

```diff
@@ -181,6 +187,19 @@ def _get_vault_credential_type():
     return models.CredentialType.objects.get(name=DefaultCredentialType.VAULT)
 
 
+def replace_vault_data(extra_var):
+    data = {
+        key: (
+            ENCRYPTED_STRING
+            if isinstance(value, str) and ANSIBLE_VAULT_STRING in value
+            else value
+        )
+        for key, value in yaml.safe_load(extra_var).items()
+    }
+
+    return yaml.safe_dump(data).rstrip("\n")
```

Comment:
@Dostonbek1 Based on the extra var discussion, we need to remove `rstrip("\n")`, and fix the e2e failures in the eda-qa side if exist